CHANGELOG.md

```diff
@@ -685,6 +685,10 @@ This section contains a list of known issues reported in Kubernetes 1.8 release.
 
 For more information, see [#53288](https://issue.k8s.io/53288).
 
+* `kubeadm init` and `kubeadm join` invocations on newly installed systems can encounter a `/var/lib/kubelet is not empty` message during pre-flight checks that prevents setup. If this is the only pre-flight failure, it can be safely ignored with `--skip-preflight-checks`.
+
```

Comment:
It would be better to recommend `kubeadm reset` before `kubeadm join` or `kubeadm init` instead of skipping checks.